core/domain/takeout_service_test.py

```diff
@@ -446,9 +447,9 @@ def set_up_trivial(self):
 
     def test_export_nonexistent_user(self):
         """Setup for nonexistent user test of export_data functionality."""
-        with self.assertRaises(
-            user_models.UserSettingsModel.EntityNotFoundError):
-            takeout_service.export_data_for_user('fake_user_id')
+        with self.assertRaises(user_models.UserSettingsModel
```

Comment:
Why are you changing the indentation from what it was before? Please keep it as it was.

Reply:
The indentation was changed for consistency with core.storage.user.gae_models_test.py, but I can change it back to what it was before.

Reply:
Done